react/src/hooks/useOnPaid.ts

```diff
@@ -0,0 +1,27 @@
+import {SendPaymentResponse} from '@webbtc/webln-types';
+import React from 'react';
+
+export function useOnPaid(
+  onPaid?: (response: SendPaymentResponse) => void,
+  payment?: SendPaymentResponse
+) {
+  React.useEffect(() => {
+    const onPaidEventHandler = (event: Event) => {
+      onPaid?.((event as CustomEvent).detail);
+    };
+    window.addEventListener('bc:onpaid', onPaidEventHandler);
+
+    if (payment) {
+      // TODO: remove once bc-send-payment accepts preimage
```

Comment:
it does in this PR, doesn't it?

Reply:
https://github.com/getAlby/bitcoin-connect/issues/162